ui/pages/home/home.component.js

```diff
@@ -156,7 +155,6 @@ export default class Home extends PureComponent {
     announcementsToShow: PropTypes.bool.isRequired,
     onboardedInThisUISession: PropTypes.bool,
     isSmartTransactionsOptInModalAvailable: PropTypes.bool.isRequired,
-    isShowTokenAutodetectModal: PropTypes.bool.isRequired,
```

Comment:
I see that this was removed from the props, but it's still being selected and passed in from the container. Can we remove it from the container as well, and delete the selector now that it's no longer used?

And on a similar note, can we consider removing the flag from state in this migration? This would resolve https://github.com/MetaMask/metamask-extension/issues/26385